recovery/elliptics_recovery/types/dc.py

```diff
@@ -234,18 +264,107 @@ def contains(addresses_with_backends, address, backend_id):
     return address_range
 
 
-def final_merge(ctx, results):
+def process_uncommitted(ctx, results):
+    '''
+    Removes uncommitted keys. If a key has any committed replicas, then this key is
+    appended to the file containing committed keys.
+    If uncommitted key's replica hasn't exceeded prepare timeout, then skip recovering of the key,
+    because the key is under writing and can be committed in the nearest future.
+    '''
+    if ctx.dry_run or ctx.safe:
+        return
+
+    elog = elliptics.Logger(ctx.log_file, int(ctx.log_level))
+    node = elliptics_create_node(address=ctx.address,
```

Comment:
`elliptics_create_node()` is a legacy method from the time there was no `elliptics.create_node()`, so it shouldn't be used in new code.